pycipapi/models.py

```diff
@@ -141,6 +141,17 @@ def __init__(self, **kwargs):
         self.cva_transaction_id = kwargs.get('cva_transaction_id')
 
 
+class ParticipantConsent(object):
+    def __init__(self, **kwargs):
+        self.created_at = kwargs.get('created_at')
+        self.updated_at = kwargs.get('updated_at')
+        self.primary_finding_consent = kwargs.get('primary_finding_consent')
+        self.carrier_status_consent = kwargs.get('carrier_status_consent')
+        self.programme_consent = kwargs.get('programme_consent')
```

Comment:
didn't we say we were removing this field?

Reply:
No we agreed to keep it with default set to `missing` didn't we? Remember I asked what's the point of having it on the table when all the values will be the same and everyone said keep it. You don't need to include it in the payload it will still work both for `post` and `put`

Reply:
ah ok yea you're right... cool that it's not required